pkg/storage/fs/nextcloud/nextcloud.go

```diff
@@ -707,7 +750,7 @@ func (nc *StorageDriver) GetQuota(ctx context.Context, ref *provider.Reference)
 	if err != nil {
 		return 0, 0, err
 	}
-	return uint64(respMap["totalBytes"].(float64)), uint64(respMap["usedBytes"].(float64)), err
+	return uint64(respMap["totalBytes"].(float64)), uint64(respMap["usedBytes"].(float64)), nil
```

Comment:
here we can define directly a struct with two fields named Quota for example:
```
type Quota struct {
  TotalBytes uint64 `json:"totalBytes"`
  UsedBytes uint64 `json:"usedBytes"`
}
```

unmarshal to this struct and return the single fields